lib/muxer.ex

```diff
@@ -9,7 +9,7 @@ defmodule Membrane.Ogg.Muxer do
   use Numbers, overload_operators: true
 
   require Membrane.Logger
-  alias Membrane.{Buffer, Opus}
+  alias Membrane.{Buffer, Ogg, Opus}
```

Comment:
Let's not alias `Membrane.Opus`, it can be confusing since we have `Membrane.Ogg.Opus` too

Reply:
by the way do you think having `Membrane.Ogg.Opus` module for Opus-specific stuff isn't too confusing?

Reply:
It's fine to me, as long as you don't alias it. But if you have an idea for a better name then let's change